### ISSUE TITLE
Template for MSbuild target / task

### ISSUE THREAD

**Issue by vlada-shubina** [dotnet/templating#4608]
Extracted from https://github.com/dotnet/templating/issues/233

Create a template for creation of MSBuild task and target file.

**Comment by vlada-shubina** [comment]
@baronfel please advise whether this issue should be moved to dotnet/msbuild repo instead.

**Comment by baronfel** [comment]
Yeah, it should. I've added a couple items to cover in the issue description. But will go ahead and transfer it.
